### ISSUE TITLE
Clearer installation directions for Windows Systems

### ISSUE THREAD

**Issue by nilajawill** [technomancy/leiningen#2564]
Hello,

Though I know that Windows installation directions have already been discussed, I can't help but see that this is still an on-going problem. The directions on Leiningen **do not** help Windows users, especially those new to Leiningen and Clojure.

I would like to present a link to a blog that details step-by-step directions for Windows users that work like a charm. I took me less than a minute to install Leiningen using the steps on the blog. Here is the link: [Installing Leiningen on Windows](https://levlaz.org/installing-leiningen-on-windows/).

Please, take the consideration of contacting the blog above to request adding the steps to the Leiningen website. There will definitely be **plenty** of appreciative people out there.

Thank you very much for your time and attention,
Nilaja D. Williams

**Comment by winks** [comment]
Hi,
I just checked the blog post and the only meaningful difference to https://leiningen.org/#install I can see are setting the PATH (which is quite different on Windows indeed, but it's not explained for other OSes either, and you can ignore the part about making it executable. Finally just running `lein` should automatically run `lein self-install`, so while the latter is more complete, it's also not strictly needed.

  a) Did you mean the part about the PATH or did I miss anything else?
  b) Are you even referring to https://leiningen.org or did you look at https://github.com/technomancy/leiningen ? Because there indeed I don't see even the `lein.bat` linked, that would explain a lot.

**Comment by technomancy** [comment]
I don't have any way of knowing what would be helpful for Windows users without more specific suggestions, so if you have some, they would be welcome.

**Comment by nilajawill** [comment]
Hello,

Thank you very much for your responses.

**Answers to the questions asked:**

- Yes, I do mean the PATH part. I definitely made a difference to have that explained. But mainly what to put in our path. 
- I am referring to https;//leiningen.org, but if this is the not correct place to add this issue, please let me know.

**Update:**

Also, I found an even better resource for Window users. There is a Leiningen installer, which does everything needed for Windows. It's much better than the blog post I shared because it gets everything working right away. Here is the link: [Leiningen Windows Installer](https://djpowell.github.io/leiningen-win-installer/).

**Purpose/Additional Information**

I am looking for more steps for Windows users because it was not very apparent on how to proper install Leiningen. And I say proper because installation is possible but there are some slight issues that happen, such as having to use "lein.bat" instead of "lein". Though I recognize that my experience installing Leiningen was from user error, I can't help but wish that there were clearer instructions on the Leiningen websites to avoid my errors. It really helps when there are concrete steps.

I really appreciate your responses. Please let me know if there is anything else I may need to add.

Thank you for very much your time and attention,
Nilaja D Williams

**Comment by nilajawill** [comment]
Hello again,

I created a repo that will contain installation instructions for various operating systems. I took in consideration that the blog I included didn't address other operating systems and am on the hunt to provide installation instructions for as many OS as possible. 

I currently only have Mac instructions at the moment but will be adding more as of today. In addition to instructions, I will also add common issues and any possible solutions as well as great resources. 

Hopefully, this will help others and answer any questions that may occur :).

The repo is here: [Leiningen Installation Instructions](https://github.com/nilajawill/leiningen-installation-instructions)

Thank you again for your time and attention! I will close this issue, but continue to add to the repo above. If you see where it can be improved, I am all ears :).
Nilaja D Williams
